eq-author/src/App/QuestionnaireDesignPage/NavigationSidebar/index.js

```diff
@@ -60,22 +60,22 @@ const OpenAllSectionsBtn = styled(Button).attrs({
   variant: "tertiary-light",
   small: true,
 })`
-  margin: 0.8em 0 0.425em 2.2em;
```

Comment:
https://www.w3schools.com/css/css_margin.asp What was wring with the shorthand?

Reply:
Nothing, I just didn't use it. I've swapped back to shorthand.